.eleventy.js

```diff
@@ -0,0 +1,10 @@
+module.exports = (eleventyConfig) => {
+  eleventyConfig.addFilter("keys", o => Object.keys(o));
+
+  return {
+    dir: {
+      input: "src",
+      output: "www",
+    }
+  };
+};
```

Comment:
Just a basic Eleventy config. THe custom filter was just for debugging, you can ✂️ it.